src/components/edit/EditMishna/ExcerptDialog.tsx

```diff
@@ -75,24 +75,14 @@ const ExcerptDialog = (props: Props) => {
       );
     }
     if (editedExcerpt.type && GROUP_NOSACH.includes(editedExcerpt.type as EXCERPT_TYPE)) {
-      return (
-        <FormikNosach
-          mishna={mishna}
-          compositions={compositions}
-          excerpt={editedExcerpt}
-          selection={selection}
-          name={'passing name'}
-          handleSubmit={(values) => console.log('handle submitting', values)}
-          handleClose={(values) => handleClose(values)}></FormikNosach>
-      );
+      return <FormNosach mishna={mishna} excerpt={editedExcerpt} selection={selection}></FormNosach>;
```

Comment:
`compositions={compositions}` is not needed for this component?

Reply:
It is not defined in the props, not used in the component, but there's no lint error in FormikNosach. am I missing something?

Reply:
I didn't check, just verifying it isn't needed